services/user.py

```diff
@@ -0,0 +1,49 @@
+from db.models import User
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> None:
+    user = User.objects.get(id=user_id)
+    if username:
+        user.username = username
+    if password:
+        user.set_password(password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```

Comment:
The `get_user` function uses `get()`, which can raise a `DoesNotExist` exception if no user is found with the given ID. Consider handling this exception to avoid potential runtime errors.